src/block/blog-posts/edit.js

```diff
@@ -945,15 +950,30 @@ export default compose(
 			order = 'desc',
 			taxonomyType = '',
 			taxonomy = '',
+			taxonomyFilterType = '__in',
 		} = props.attributes
 		const { getEntityRecords } = select( 'core' )
 
+		const categoryQuery = {
+			categories: ( taxonomyFilterType === '__in' && taxonomyType === 'category' && taxonomy ) ?
+				taxonomy :
+				undefined,
+			categories_exclude: taxonomyFilterType === '__not_in' && taxonomyType === 'category' && taxonomy ? taxonomy : undefined, // eslint-disable-line camelcase
+		}
```

Comment:
Just made both lines similar

```suggestion
		const categoryQuery = {
			categories: taxonomyFilterType === '__in' && taxonomyType === 'category' && taxonomy ? taxonomy : undefined,
			categories_exclude: taxonomyFilterType === '__not_in' && taxonomyType === 'category' && taxonomy ? taxonomy : undefined, // eslint-disable-line camelcase
		}
```